src/config/dynamic-theme-fixes.config

```diff
@@ -28353,3 +28353,13 @@ CSS
 .zyimage {
     background-color: var(--darkreader-neutral-text) !important;
 }
+
+================================
+
+center-pf.kakao.com
+
+CSS
+.box_bubble .txt_chat {
+    color: rgb(255 255 255);
```

Comment:
As I can not open myself this page I would like you to check how this will work with Light scheme set to dark reader. Because I think it should use black color then.

To fix that use:
`color: ${rgb(0 0 0)};`

It will make text white in Dark scheme and black in light.

Reply:
In a bright scheme, it appears as shown above. (default)
![2024-02-20 14 08 05](https://github.com/darkreader/darkreader/assets/92414961/3ef437f2-4a4e-46e2-973c-07c40256a4a0)

to fix color: `${rgb(0 0 0)}`
![2024-02-20 14 10 50](https://github.com/darkreader/darkreader/assets/92414961/09dc2eba-5496-4f62-8c7a-46baf5764087)